package.json

```
@@ -6,7 +6,8 @@
   "scripts": {
     "test": "npm run lint && npm run unit-tests",
     "lint": "eslint .",
-    "unit-tests": "mocha ./tests/**/*.js"
+    "unit-tests": "mocha ./tests/**/*.js",
+    "clean": "rm -rf ./generated/*"
```

```suggestion
    "clean": "rm -rf ./generated/*.json"
```

rm -rf is kinda scary, let's limit to json if possible

---

We also have some `.txt` files in this directory.

---

How about `rm -rf ./generated/*.{json,txt}`?